standalone/pruntime/enclave/src/contracts/btc_lottery.rs

```diff
@@ -494,17 +489,22 @@ impl contracts::Contract<Command, Request, Response> for BtcLottery {
             {
                 Self::open_lottery(self, round_id, token_id, btc_address);
             } else if let chain::pallet_bridge_transfer::Event::LotteryPayloadSend(
-                payload_type,
                 chain_id,
                 payload,
                 sequence,
             ) = pe
             {
+                let payload_data = match LotteryPayload::decode(&mut &payload[..]) {
+                    Ok(p) => p,
+                    Err(err) => {
+                        error!("Decode of payload: {:?} crashes!", payload);
+                        return;
+                    }
+                };
```

Comment:
I think only sequence is used in this event. Why do we decode the entire payload?